packages/neovim.rb

```diff
@@ -3,25 +3,26 @@
 class Neovim < Package
   description 'Neovim is a refactor, and sometimes redactor, in the tradition of Vim (which itself derives from Stevie).'
   homepage 'https://neovim.io/'
-  version '0.2.0'
-  source_url 'https://github.com/neovim/neovim/archive/v0.2.0.tar.gz'
-  source_sha256 '72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5'
+  version '0.4.2'
+  source_url 'https://github.com/neovim/neovim/archive/v0.4.2.tar.gz'
+  #source_sha256 '72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5'
+  source_sha256 '9f874d3d2a74f33b931db62adebe28f8d2ec116270d1e13998b58a73348b6e56'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-x86_64.tar.xz',
+#    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-armv7l.tar.xz',
+#     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-armv7l.tar.xz',
+#       i686: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-i686.tar.xz',
+#     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/neovim-0.2.0-chromeos-x86_64.tar.xz',
```

Comment:
Please remove the lines instead of commenting them.

Reply:
Corrected in commit 495c345